src/go/k8s/cmd/configurator/main.go

```diff
@@ -131,9 +131,11 @@ func main() {
 
 	cfg.Redpanda.ID = int(hostIndex)
 
-	// First Redpanda node need to have cleared seed servers in order
-	// to form raft group 0
-	if hostIndex == 0 {
+	// In case of a single seed server, the list should contain the current node itself.
+	// Normally the cluster is able to recognize it's talking to itself, except when the cluster is
+	// configured to use mutual TLS.
+	// So, we clear the list of seeds to help Redpanda.
```

Comment:
do you know why is that? the nodes does not talk to itself by mtls. Is this a bug in redpanda they are looking to fix at one point?

Reply:
https://github.com/redpanda-data/redpanda/issues/333#issuecomment-1153927714